package br.com.tln.personalcard.usuario.exception

class ConnectionErrorException(cause: Throwable) : Exception(cause)